docs/actions-scripts.md

```diff
@@ -33,17 +112,17 @@ We currently package version 16.1, with `datacheck`, `safetab`, and `safecount`
 
 As Stata is a commercial product, a license key is needed to use it. 
 
-#### If you are a member of the opensafely GitHub organisation
+#### If you are a member of the OpenSAFELY GitHub organisation
 * If you are using Windows, then the `opensafely` command line software will
-automatically use the OpenSAFELY stata license. 
-* If you are using macOS:
+automatically use the OpenSAFELY Stata license. 
+* If you are using MacOS:
    1. Download and install [GitHub's command-line tool (`gh`)](https://cli.github.com/)
```

Comment:
these are rendering as bullets not numbered

Reply:
I didn't intentially change this, will revert